### PR TITLE
Add a /api/users/roles endpoint and use it in the UI

### PR DESCRIPTION
**Pull request by allysonjp715**

### CODE REVIEW

File: verification/curator-service/ui/src/components/Users.test.tsx [hunk 1]
```diff
@@ -37,7 +37,14 @@ test('lists users', async () => {
         config: {},
         headers: {},
     };
-    mockedAxios.get.mockResolvedValueOnce(axiosResponse);
+    mockedAxios.get.mockImplementation((url) => {
+        if (url === '/api/users/roles') {
```
Comment (attwad) [inline verification/curator-service/ui/src/components/Users.test.tsx:41]: can you use a switch on the url please and default to reject the promise? this way we're sure we're mocking the right calls.
switch (url) {
  case roles: resolve roles
  case users: resolve users
  default: reject promise
}

File: verification/curator-service/ui/src/components/Users.test.tsx [hunk 2]
```diff
@@ -37,7 +37,14 @@ test('lists users', async () => {
...
+            return Promise.resolve({
+                data: { roles: ['admin', 'curator', 'reader'] },
```
Comment (attwad) [inline verification/curator-service/ui/src/components/Users.test.tsx:43]: this promise data is similar in all mocked calls, consider refactoring it

File: verification/curator-service/ui/src/components/Users.test.tsx (hunk 1 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/Users.test.tsx]: Done

File: verification/curator-service/ui/src/components/Users.test.tsx (hunk 2 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/Users.test.tsx]: Done

**Pull request merged**